contents/articles/2016-11_facevr/index.md

```diff
@@ -0,0 +1,26 @@
+--- 
+title: "FaceVR: Real-Time Facial Reenactment and Eye Gaze Control in VR"
+date: Thu Nov 17 2016 10:49:16 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "A novel approach for real-time real-time gaze-aware facial performance capture system to drive the photo-realistic reconstructed digital face in virtual reality..."
+featured_img: "../../../../media/images/blog/face_vrb.png"
+featured_img_thumb: "../../../../mesdia/images/blog/thumb/face_vr.png"
+---
+
+<img class="Feature-image u-padBottom--1" src="../../../../media/images/blog/face_vr.png" alt="Real-Time Facial Reenactment and Eye Gaze Control in VR">
+
+Source: [FaceVR: Real-Time Facial Reenactment and Eye Gaze Control in VR](https://arxiv.org/pdf/1610.03151.pdf)
+
+<br>
+
+[Justus Thies](http://lgdv.cs.fau.de/people/card/justus/thies/) et al., presents a novel approach for real-time gaze-aware facial capture system to drive the photo-realistic reconstructed digital face in virtual reality. A facial reenactment that can transfer facial expressions and realistic eye appearance between a source and a target actor video.
+
+Head-mounted displays (HMD) are able to provide immersive renderings of virtual environments. But while they are able to provide very believable immersive renderings, the headset themselves occludes the majority of the face. To overcome this, a 3D face is constructed using an RGB-D camera to capture the facial performance of the source actor and [Pupil's Oculus Rift DK2 add-on cup](https://pupil-labs.com/store/#vr-ar) was used to track the actor's eye within the HMD. The source actor's facial and gaze data is then used to drive the photo-realistic facial animations of the target video, therefore enabling gaze-aware facial reenactment. 
```

Comment:
Head mounted displays (HMDs) provide immersive renderings of virtual environments. But in order to do so HMDs block the majority of the actor/participant's face. In order to reconstruct the actor/participant's face, Thies et al., use an RGB camera to capture the facial performance of the participant and Pupil Labs's [Oculus Rift DK2 add-on cup] to capture eye movements within the HMD. The source actor's facial and eye movement data is then used to drive the photo-realistic..."